04_live_update/Tiltfile

```diff
@@ -0,0 +1,24 @@
+docker_build("frontend", ".", dockerfile="frontend.dockerfile",
+  live_update=[
+    sync('./frontend', '/app/frontend'),
```

Comment:
Won't this cause the old binary to be synced *and then* the new binary gets compiled? Or do things not happen in the order they appear here?

Reply:
the idea with this is that we never build the binary locally - we're only syncing code

Reply:
Ohhh, this runs inside the container—got it.